lib/Checks.groovy

```diff
@@ -1,9 +1,29 @@
+import org.yaml.snakeyaml.Yaml
+
 /*
  * This file holds several functions used to perform standard checks for the nf-core pipeline template.
  */
 
 class Checks {
 
+    static void check_conda_channels() {
+        def message = "There's a problem with your conda configuration. " +
+                      "Please refer to  https://bioconda.github.io/user/install.html#set-up-channels " + 
+                      "to learn how to set-up the conda-forge and bioconda channels correctly. "
+                      "NB: the order of the channels matters!"
+
+        Yaml parser = new Yaml()
+        def channels = parser.load("conda config --show channels".execute().text).channels
+
+        // check that all channels are present
+        def required_channels = ['conda-forge', 'bioconda', 'defaults']
+        required_channels.each { ch -> assert ch in channels: message }
```

Comment:
Will these `assert` statements cause a hard failure? I wonder if we warn for now instead until this has been tested properly in the real world?

Reply:
Makes sense :+1: